modules/deployment-plug-in-cluster.adoc

```diff
@@ -10,32 +10,79 @@ After pushing an image with your changes to a registry, you can deploy the plug-
 
 .Procedure
 
-. To deploy your plug-in to a cluster, instantiate your plug-in by running the following command:
+. To deploy your plug-in to a cluster, install a Helm chart with the name of the plug-in as the Helm release name into a new namespace or an existing namespace as specified by the `-n` command-line option. Provide the location of the image within the `plugin.image` parameter by using the following command:
+
 +
 [source,terminal]
 ----
-$ oc process -f template.yaml \
-  -p PLUGIN_NAME=my-plugin \ <1>
-  -p NAMESPACE=my-plugin-namespace \ <2>
-  -p IMAGE=quay.io/my-repository/my-plugin:latest \ <3>
-  | oc create -f -
+helm upgrade -i  my-plugin charts/openshift-console-plugin -n my-plugin-namespace --create-namespace --set plugin.image=my-plugin-image-location
 ----
-<1> Update with the name of your plug-in.
-<2> Update with the namespace.
-<3> Update with the name of the image you created.
-+
-This command runs a light-weight NGINX HTTP server to serve the assets for your plug-in.
 
-[IMPORTANT]
-====
-`PLUGIN_NAME` must match the plug-in name you used in the `consolePlugin` declaration of `package.json`.
-====
+. Optional: You can specify any additional parameters by using the values set of supported parameters in the `charts/openshift-console-plugin/values.yaml` file.
```

Comment:
_values set of_ doesn't sound right to me. 

_by using the set of supported parameters_ ??

Reply:
A member of the target audience might know this, but how does one specify the parameters? Edit the YAML file or append them to the `helm upgrade` command? 
Do we need to state where the file comes from or what it is? Maybe it's not needed, but I like the description [here](https://docs.openshift.com/container-platform/4.11/operators/operator_sdk/helm/osdk-helm-tutorial.html#osdk-helm-modify-cr_osdk-helm-tutorial).